pkgs/development/python-modules/emv/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub
+, click, enum-compat, pyscard, pycountry, terminaltables
+, pytestCheckHook, unittest2
```

Comment:
```suggestion
, pytestCheckHook
```

Reply:
Dropped now this is updated to 1.0.11 (note: you were looking at master, which was quite a bit after the 1.0.9 release I'd initially packaged)